packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -332,41 +331,6 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
       },
       async write(chunk: IAsset, _encoding, callback) {
         await transaction?.attach(async () => {
-          // TODO: Remove this logic in V5
```

Comment:
Should we add a warning with a clear description that if the transfer fails due to missing metadata, it might be because they're trying to use an outdated dataset?

Reply:
Might be worth thinking about (for a future task/PR) a small script to transform old datasets into new ones (basically creating metadata out of the file entities)

Reply:
Just to be sure this couldn't be considered a breaking change, did v5 every export without metadata?

I think considering we don't officially support v4 backups imported into v5, it's fine to break those cases.

Reply:
yep every single v5 exports metadata as well as every latest v4, this was a very old fix